Makefile

```diff
@@ -58,6 +58,8 @@ publish:
 bootstrap:
 	npm install
 	./node_modules/.bin/lerna bootstrap
+	# remove all existing babel-runtimes and use the top-level babel-runtime
+	rm -rf packages/*/node_modules/babel-runtime
```

Comment:
I think we could do a PR to lerna later so we can get rid of this (modify bootstrap to ignore modules in a config)